config.ini.in

```diff
@@ -92,7 +92,6 @@ cache_file_max = 50000
 # and {CONF_PATH} with the path to this config file
 compiler_cmd = "${VE_OPENMP_COMPILER_CMD} ${VE_OPENMP_COMPILER_FLG} ${VE_OPENMP_COMPILER_INC} ${VE_OPENMP_COMPILER_LIB} {IN} -o {OUT}"
 # JIT compile options
-compiler_openmp = ${_VE_OPENMP_COMPILER_OPENMP}
```

Comment:
Why? I like that I can enable OpenMP but not use OpenMP's SIMD

Reply:
It got chopped in my first iteration and didn't make the cut, when I put back stuff again. I have reinserted it again.